.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -122,28 +126,27 @@ In order to avoid last minute blockers and significant disruptions to the featur
 
 ### **Legal** (Complete before Feature Complete Date)
 - [ ] Changed or new open source libraries are cleared and approved, or N/A. (Legal Release Services/Cass Tucker/Release PM).
-- [ ] Licenses and Certificates of Originality (COOs) are updated, or N/A
-
-### **Translation** (Complete 1 week before Feature Complete Date)
-- [ ] PII updates are merged, or N/A. Note timing with translation shipments.
 
 ### **Innovation** (Complete 1 week before Feature Complete Date)
 - [ ] Consider whether any aspects of the feature may be patentable. If any identified, disclosures have been submitted.
 
+### **Translation** (Complete by Feature Complete Date)
+- [ ] PII (Program Integrated Information) updates are merged (i.e. all English strings due for translation have been delivered), or N/A.
```

Comment:
That's only if we needed the content to come back translated prior to the feature going GA.  Since that's no longer the case (for `Translation - Missing`), then as long as their strings are delivered (just like the rest of their code), they're ok.
Also, I'm hopeful that in the not-to-distant future, this process will be completely automated and done on each release build, rather then a select set of builds only.  

But @pnickoll in case he thinks the above needs to be changed